src/ifvc.c

```diff
@@ -274,6 +281,21 @@ void buildIfVc(void) {
             mask = s_addr_from_sockaddr(&IfReq.ifr_addr); // Do not use ifr_netmask as it is not available on freebsd
             subnet = addr & mask;
 
+            dp = getIfByName(IfPt->ifr_name, 1);
```

Comment:
What is the purpose of function parameter which is always constant?

Reply:
It is to be able to take advantage of the function so that it also works in the same way as it did in the past.